frontend/server/src/Controllers/Contest.php

```diff
@@ -98,6 +99,14 @@ public static function apiList(\OmegaUp\Request $r): array {
         $contests = [];
         $r->ensureOptionalInt('page');
         $r->ensureOptionalInt('page_size');
+        \OmegaUp\Validators::validateStringOfLengthInRange(
```

Comment:
mejor hay que usar [`ensureOptionalEnum`](https://github.com/omegaup/omegaup/blob/21ee8588fea5489fe9f0ecdd5ba8353ef1e24875/frontend/server/src/Request.php#L409):

```suggestion
        $tabName = $r->ensureOptionalEnum(
```